<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="PMainEsquemaUI" name="mainEsquemaUI" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_rowFormat"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1343</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_NewEsquema"/>
    <addaction name="action_LoadSession"/>
    <addaction name="action_SaveSession"/>
    <addaction name="action_SaveSessionAs"/>
    <addaction name="separator"/>
    <addaction name="action_ExportEsquema"/>
    <addaction name="action_ImportEsquema"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_fileBrowser">
   <property name="windowTitle">
    <string>File Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="WBrowserTreeView" name="browserWidget">
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="indentation">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_changeRoot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Change Root</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_NewEsquema">
   <property name="text">
    <string>New Esquema</string>
   </property>
  </action>
  <action name="action_LoadSession">
   <property name="text">
    <string>Load Session</string>
   </property>
  </action>
  <action name="action_SaveSession">
   <property name="text">
    <string>Save Session</string>
   </property>
  </action>
  <action name="action_ExportEsquema">
   <property name="text">
    <string>Export Esquema</string>
   </property>
  </action>
  <action name="action_ImportEsquema">
   <property name="text">
    <string>Import Esquema</string>
   </property>
  </action>
  <action name="action_SaveSessionAs">
   <property name="text">
    <string>Save Session As...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WBrowserTreeView</class>
   <extends>QTreeView</extends>
   <header>WBrowserTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>PMainEsquemaUI</class>
   <extends>QWidget</extends>
   <header>PMainEsquemaUI.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
